core/aws_ddk_core/stages/lambda_handlers/appflow_check_flow_status/index.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
So I suppose this will be renamed to `lambda_function.py`?

Reply:
yes - I've just renamed it.